test/AutoRest.TestServerLowLevel.Tests/LowLevel/Generation/ReadonlyPropertyWritingTests.cs

```diff
@@ -177,6 +191,18 @@ public partial class RoundTripModel: global::Azure.Core.IUtf8JsonSerializable
 optionalReadonlyInt = property.Value.GetInt32();
 continue;
 }
+if(property.NameEquals(""requiredReadonlyModel"")){
+requiredReadonlyModel = global::Cadl.TestServer.ReadonlyProperties.Models.ReadonlyModel.DeserializeReadonlyModel(property.Value);
+continue;
+}
+if(property.NameEquals(""optionalReadonlyModel"")){
+if (property.Value.ValueKind == global::System.Text.Json.JsonValueKind.Null)
+{
+property.ThrowNonNullablePropertyIsNull();
+continue;}
```

Comment:
why we need to continue after throw the exception?

Reply:
@lirenhe This is a debug assertion. See https://github.com/Azure/autorest.csharp/blob/795b9d22788b838f8064d40f0dd7e7d7ebe744b1/src/assets/Generator.Shared/JsonElementExtensions.cs#L94-L99